packages/sit-onyx/src/components/OnyxSlider/OnyxSlider.vue

```diff
@@ -0,0 +1,725 @@
+<script setup lang="ts">
+import { createSlider } from "@sit-onyx/headless";
+import { iconMinusSmall, iconPlusSmall } from "@sit-onyx/icons";
+import { computed, toRefs } from "vue";
+import { useDensity } from "../../composables/density.js";
+import { getFormMessages, useCustomValidity } from "../../composables/useCustomValidity.js";
+import { useErrorClass } from "../../composables/useErrorClass.js";
+import {
+  SKELETON_INJECTED_SYMBOL,
+  useSkeletonContext,
+} from "../../composables/useSkeletonState.js";
+import { useVModel } from "../../composables/useVModel.js";
+import { useRootAttrs } from "../../utils/attrs.js";
+import { FORM_INJECTED_SYMBOL, useFormContext } from "../OnyxForm/OnyxForm.core.js";
+import OnyxFormElement from "../OnyxFormElement/OnyxFormElement.vue";
+import OnyxIconButton from "../OnyxIconButton/OnyxIconButton.vue";
+import OnyxSkeleton from "../OnyxSkeleton/OnyxSkeleton.vue";
+import OnyxTooltip from "../OnyxTooltip/OnyxTooltip.vue";
+import type { OnyxSliderProps } from "./types.js";
+
+const clamp = (value: number, min: number, max: number) => Math.min(Math.max(value, min), max);
```

Comment:
There is already a `applyLimits` utility for this that you can re-use / import from "utils/numbers.ts"